### ISSUE TITLE
API error caused by JSON helper

### ISSUE THREAD

**Issue by jmauzyk** [verbb/formie#1414]
### Describe the bug

Getting the following error when a payload is sent to Freshdesk and a contact already exists:

```log
2023-05-02 09:51:37 [ERROR] Freshdesk: API error: “Attempt to read property "errors" on array” /home/user/craft/vendor/verbb/formie/src/integrations/crm/Freshdesk.php:378
```

On `Freshdesk.php` line 378, `Json::decode` needs a second param set to `false`. Without this, the default behavior for Yii's helper is to [return an array](https://github.com/yiisoft/yii2/blob/master/framework/helpers/BaseJson.php#L120), which results in the error above when attempting to read `errors` as an object property.

### Steps to reproduce

1. On a form with Freshdesk integration enabled and contact field mapping set up, send an integration payload including an email for an existing contact.
2. Produces error above.

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.4.8

### Plugin version

2.0.28

### Multi-site?

Yes

**Comment by engram-design** [comment]
Thanks, just FYI this was code from your own [PR](https://github.com/verbb/formie/pull/827), but I did swap `json_decode` with `Json::decode` for consistency without checking for object syntax. 

Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.28`

**Comment by jmauzyk** [comment]
I see, I missed that arrays were used for other integrations. I'll update Freshdesk for consistency and open a PR sometime. In the meantime, thanks for the quick fix!

**Comment by engram-design** [comment]
Fixed in [2.0.29](https://github.com/verbb/formie/releases/tag/2.0.29)
